pyproject.toml

```diff
@@ -1,7 +1,105 @@
+[project]
+name = "extension-helpers"
+authors = [{name = "The Astropy Developers", email = "astropy.team@gmail.com"}]
+license = {text = "BSD 3-Clause License"}
+description = "Utilities for building and installing packages with compiled extensions"
+readme = "README.rst"
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Intended Audience :: Developers",
+    "Framework :: Setuptools Plugin",
+    "License :: OSI Approved :: BSD License",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+    "Topic :: Software Development :: Build Tools",
+    "Topic :: Software Development :: Libraries :: Python Modules",
+    "Topic :: System :: Archiving :: Packaging",
+]
+requires-python = ">=3.7"
+dependencies = [
+    "setuptools>=40.2",
+    "tomli>=1.0.0",
+]
+dynamic = ["version"]
+
+[project.urls]
+Homepage = "https://github.com/astropy/extension-helpers"
+
+[project.entry-points."setuptools.finalize_distribution_options"]
+extension_helpers_get_extensions = "extension_helpers:_finalize_distribution_hook"
+
+[project.optional-dependencies]
+test = [
+    "wheel",
+    "pytest",
+    "pytest-cov",
+    "pytest-astropy",
```

Comment:
Are you sure you need pytest-astropy here? Doesn't that pull in astropy as a test dependency?

Reply:
Fixed/removed